scripts/monitor_dbus_signals.py

```diff
@@ -151,6 +155,8 @@ def _properties_changed(*props_changed, object_path=None):
                 for prop in properties_invalidated:
                     data[interface_name][prop] = INVALIDATED
 
+        print("Properties changed.")
```

Comment:
Indent four spaces...needs to be emitted when ```_properties_changed()``` is invoked.

Reply:
OK; after that change, I now see this:
```
$ sudo stratis pool create spool1 /dev/vdb1

Interfaces added.
Interfaces added.


$ sudo stratis fs create spool1 sfs1

Interfaces added.
Properties changed.



$ sudo stratis fs destroy spool1 sfs1
Interfaces removed.
Properties changed.


$ sudo stratis pool destroy spool1

Interfaces removed.
Interfaces removed.
```